pollination_streamlit/api/recipes.py

```diff
@@ -14,3 +14,16 @@ def get_recipe(self, owner: str, name: str, tag: str = 'latest') -> Recipe:
             path=f'/registries/{owner}/recipe/{name}/{tag}/json'
         )
         return Recipe.parse_obj(res)
+
+    def set_project(self, owner: str, name: str,
```

Comment:
Can we rename this to `add_to_project`? The recipe doesn't have a setter for the project. Instead, it is being added to a project.